.gitignore

```diff
@@ -76,4 +76,7 @@ build/
 !**/ios/**/default.perspectivev3
 
 # Tizen Rootstraps
-rootstraps/
\ No newline at end of file
+rootstraps/
+
+# Docs
+docs/**/*.png
```

Comment:
There are only `.drawio` files in the directory. How do they work?

Reply:
The .drawio files are just diagrams that can be viewed with https://draw.io.

Reply:
https://app.diagrams.net/#HWonyoungChoi%2Ftizen_interop%2Fadd_6.5_bindings%2Fdocs%2Fdiagrams%2Fmodule_view.drawio

Reply:
Will the diagrams be used anywhere in doc or are they just for who are interested?

Reply:
Only people interested in "internal module view".